packages/fuchsia_ctl/lib/src/tar.dart

```diff
@@ -42,9 +42,9 @@ class SystemTar implements Tar {
   ) async {
     final ProcessResult result = await processManager.run(<String>[
       'tar',
-      '-xvf', src, //
+      '-xf', src, //
       '-C', destination,
-    ]);
+    ]).timeout(const Duration(minutes: 10));
```

Comment:
Do we know how long this takes on average? Would it really ever come close to 5?  This seems long.

Reply:
Tested it on my machine it is ~0.5 mins setting the default to 5.

> Test for the tar.dart change?

Done